CAAppKitBridge/Demo/DemoController.m

```diff
@@ -0,0 +1,65 @@
+#import <AppKit/NSWindow.h>
+#import <AppKit/NSOpenGL.h>
+#import <AppKit/NSApplication.h>
+#import <AppKit/NSMenu.h>
+#import <CAAppKitBridge/NSView+CAmethods.h>
```

Comment:
1. This header does not exist until installation, so the demo won't build if built as a subproject of the root project.
2. This header will not exist even after installation, as the header is now called `NSView+CAMethods.h`.
3. This header will not exist even after installation, as the header is no longer installable (it's a private header).
4. The header is not supposed to be installable; no additional methods should be exposed that are not already visible from `-gui`'s `NSView.h`.

Reply:
1. True
2. True
3, 4. - are you sure? For example I couldn't find wantsLayer in NSView.h
Lets assume you are right, what do you propose then? How should I import the header, if it should be imported at all?

Reply:
> 3, 4. - are you sure?

Yes. :)

> For example I couldn't find wantsLayer in NSView.h

This would suggest you should add it to NSView.h, as well as a stub implementation to NSView.m, which checks if the real implementation exists, and then calls it.

> Lets assume you are right, what do you propose then? How should I import the header, if it should be imported at all?

It should not be imported because it does not exist installed on a system (unless you have a very, very strong reason why the source code of the application -- not just the build system -- should need to unavoidably be aware that it's running on GNUstep, and only then import the required header).  :)